sentry/src/transports/tokio_thread.rs

```diff
@@ -85,7 +85,12 @@ impl TransportThread {
     }
 
     pub fn send(&self, envelope: Envelope) {
-        let _ = self.sender.send(Task::SendEnvelope(envelope));
+        // Using send here would mean that when the channel fills up for whatever
```

Comment:
In JS we indeed just drop the envelope - https://github.com/getsentry/sentry-javascript/blob/e497bcda305a9c248dcdc78f48fe548cc58f8233/packages/core/src/transports/base.ts#L93-L104